udata_front/theme/gouvfr/datagouv-components/src/components/AppLink/AppLink.vue

```diff
@@ -18,8 +18,10 @@ withDefaults(defineProps<RouterLinkProps>(), {
 });
 const router = inject(routerKey, null);
 const attrs = useAttrs();
-const externalAttrs = !router ? { ...attrs, target: attrs.target || "_blank" } : {};
+const externalAttrs = { ...attrs, ...( !router ? { target: attrs.target } : {} ) };
```

Comment:
```suggestion
const externalAttrs = { ...attrs, target: router ? "_blank" : attrs.target };
```
When we have a router, in the isExternalLink case, we want to force the `_blank`. Otherwise, we want to use the provided `attrs.target`.